lib/crypto/src/crypto.app.src

```diff
@@ -17,14 +17,15 @@
 %%
 %% %CopyrightEnd%
 %%
-{application, crypto,
-   [{description, "CRYPTO"},
+{application, crypto, [
+    {description, "CRYPTO"},
     {vsn, "%VSN%"},
-    {modules, [crypto,
-	       crypto_ec_curves]},
+    {modules, [
```

Comment:
Does it do this only if "not in same line" (as it the case with :28, below)?

Reply:
It makes sense. I guess, in the future, in any case, there'll probably be more options to adapt to certain "preferences" (e.g. if they're widespread).

Reply:
`erlfmt` is an opinionated formatter.  We have very limited configuration and we intend to keep it this way.
We are trying to reach a format that most erlang developers can conform to.
Not saying this is going to be easy, but lots of languages have achieved this, but retro fitting a formatter onto an existing community is going to be even tougher.
Either way it would be great if we could talk through our different opinions and assume we can't configure them later.

Reply:
Fair enough. It's your call in the end, in any case 😄